### ISSUE TITLE
Podspec for alpha2

### ISSUE THREAD

**Issue by bjarkehs** [ashfurrow/ReactiveCocoa#2]
Hey here's the podspec: https://github.com/bjarkehs/ReactiveCocoa/blob/v4.0-alpha.2-podspec/ReactiveCocoa.podspec

But I ran into the following error:
```
** BUILD SUCCEEDED **

 -> ReactiveCocoa (4.0.0-alpha-2)
    - WARN  | source: The version should be included in the Git tag.
    - ERROR | [ReactiveCocoa/UI] xcodebuild: Returned an unsuccessful exit code.
    - NOTE  | [ReactiveCocoa/UI] xcodebuild:  <module-includes>:1:9: note: in file included from <module-includes>:1:
    - NOTE  | [ReactiveCocoa/UI] xcodebuild:  Target Support Files/ReactiveCocoa/ReactiveCocoa-umbrella.h:75:9: note: in file included from Target Support Files/ReactiveCocoa/ReactiveCocoa-umbrella.h:75:
    - ERROR | [ReactiveCocoa/UI] xcodebuild:  ReactiveCocoa/ReactiveCocoa/ReactiveCocoa.h:67:10: error: include of non-modular header inside framework module 'ReactiveCocoa.ReactiveCocoa'
    - ERROR | [ReactiveCocoa/UI] xcodebuild:  ReactiveCocoa/ReactiveCocoa/ReactiveCocoa.h:68:10: error: include of non-modular header inside framework module 'ReactiveCocoa.ReactiveCocoa'
    - ERROR | [ReactiveCocoa/UI] xcodebuild:  ReactiveCocoa/ReactiveCocoa/ReactiveCocoa.h:69:10: error: include of non-modular header inside framework module 'ReactiveCocoa.ReactiveCocoa'
    - ERROR | [ReactiveCocoa/UI] xcodebuild:  ReactiveCocoa/ReactiveCocoa/ReactiveCocoa.h:70:10: error: include of non-modular header inside framework module 'ReactiveCocoa.ReactiveCocoa'
    - ERROR | [ReactiveCocoa/UI] xcodebuild:  ReactiveCocoa/ReactiveCocoa/ReactiveCocoa.h:71:10: error: include of non-modular header inside framework module 'ReactiveCocoa.ReactiveCocoa'
    - ERROR | [ReactiveCocoa/UI] xcodebuild:  ReactiveCocoa/ReactiveCocoa/ReactiveCocoa.h:72:10: error: include of non-modular header inside framework module 'ReactiveCocoa.ReactiveCocoa'
    - ERROR | [ReactiveCocoa/UI] xcodebuild:  ReactiveCocoa/ReactiveCocoa/ReactiveCocoa.h:73:10: error: include of non-modular header inside framework module 'ReactiveCocoa.ReactiveCocoa'
    - ERROR | [ReactiveCocoa/UI] xcodebuild:  ReactiveCocoa/ReactiveCocoa/ReactiveCocoa.h:74:10: error: include of non-modular header inside framework module 'ReactiveCocoa.ReactiveCocoa'
    - ERROR | [ReactiveCocoa/UI] xcodebuild:  ReactiveCocoa/ReactiveCocoa/ReactiveCocoa.h:75:10: error: include of non-modular header inside framework module 'ReactiveCocoa.ReactiveCocoa'
    - ERROR | [ReactiveCocoa/UI] xcodebuild:  ReactiveCocoa/ReactiveCocoa/ReactiveCocoa.h:76:10: error: include of non-modular header inside framework module 'ReactiveCocoa.ReactiveCocoa'
    - ERROR | [ReactiveCocoa/UI] xcodebuild:  ReactiveCocoa/ReactiveCocoa/ReactiveCocoa.h:77:10: error: include of non-modular header inside framework module 'ReactiveCocoa.ReactiveCocoa'
    - ERROR | [ReactiveCocoa/UI] xcodebuild:  ReactiveCocoa/ReactiveCocoa/ReactiveCocoa.h:78:10: error: include of non-modular header inside framework module 'ReactiveCocoa.ReactiveCocoa'
    - ERROR | [ReactiveCocoa/UI] xcodebuild:  ReactiveCocoa/ReactiveCocoa/ReactiveCocoa.h:79:10: error: include of non-modular header inside framework module 'ReactiveCocoa.ReactiveCocoa'
    - ERROR | [ReactiveCocoa/UI] xcodebuild:  ReactiveCocoa/ReactiveCocoa/ReactiveCocoa.h:80:10: error: include of non-modular header inside framework module 'ReactiveCocoa.ReactiveCocoa'
    - ERROR | [ReactiveCocoa/UI] xcodebuild:  ReactiveCocoa/ReactiveCocoa/ReactiveCocoa.h:81:10: error: include of non-modular header inside framework module 'ReactiveCocoa.ReactiveCocoa'
    - ERROR | [ReactiveCocoa/UI] xcodebuild:  ReactiveCocoa/ReactiveCocoa/ReactiveCocoa.h:82:10: error: include of non-modular header inside framework module 'ReactiveCocoa.ReactiveCocoa'
    - ERROR | [ReactiveCocoa/UI] xcodebuild:  ReactiveCocoa/ReactiveCocoa/ReactiveCocoa.h:83:10: error: include of non-modular header inside framework module 'ReactiveCocoa.ReactiveCocoa'
    - ERROR | [ReactiveCocoa/UI] xcodebuild:  ReactiveCocoa/ReactiveCocoa/ReactiveCocoa.h:84:10: error: include of non-modular header inside framework module 'ReactiveCocoa.ReactiveCocoa'
    - ERROR | [ReactiveCocoa/UI] xcodebuild:  ReactiveCocoa/ReactiveCocoa/ReactiveCocoa.h:85:10: error: include of non-modular header inside framework module 'ReactiveCocoa.ReactiveCocoa'
    - NOTE  | [ReactiveCocoa/UI] xcodebuild:  <unknown>:0: error: could not build Objective-C module 'ReactiveCocoa'

Analyzed 2 podspecs.

[!] 1 out of 2 specs failed validation.

/Users/bjarkehs/.rvm/gems/ruby-2.2.0@global/gems/cocoapods-0.39.0/lib/cocoapods/command/spec/lint.rb:77:in `run'
/Users/bjarkehs/.rvm/gems/ruby-2.2.0/gems/claide-0.9.1/lib/claide/command.rb:312:in `run'
/Users/bjarkehs/.rvm/gems/ruby-2.2.0@global/gems/cocoapods-0.39.0/lib/cocoapods/command.rb:47:in `run'
/Users/bjarkehs/.rvm/gems/ruby-2.2.0@global/gems/cocoapods-0.39.0/bin/pod:44:in `<top (required)>'
/Users/bjarkehs/.rvm/gems/ruby-2.2.0/bin/pod:23:in `load'
/Users/bjarkehs/.rvm/gems/ruby-2.2.0/bin/pod:23:in `<main>'
/Users/bjarkehs/.rvm/gems/ruby-2.2.0/bin/ruby_executable_hooks:15:in `eval'
/Users/bjarkehs/.rvm/gems/ruby-2.2.0/bin/ruby_executable_hooks:15:in `<main>'
```
I read this http://stackoverflow.com/questions/24103169/swift-compiler-error-non-modular-header-inside-framework-module but not sure how to properly apply it in a podspec. Any ideas?

**Comment by bjarkehs** [comment]
I realize this issue was there with the previous podspec too. So I guess not at present time. Hmm...

**Comment by ashfurrow** [comment]
This is a pretty big issue – not sure of the fix, since I'm sticking on Xcode 7.0 for now (saw this with 7.1). [This thread](https://github.com/CocoaPods/CocoaPods/issues/4056) has some possible fixes.

**Comment by ashfurrow** [comment]
Should be fixed: https://github.com/CocoaPods/Specs/blob/master/Specs/ReactiveCocoa/4.0.0-alpha-2/ReactiveCocoa.podspec.json Please re-open if it doesn't work.
